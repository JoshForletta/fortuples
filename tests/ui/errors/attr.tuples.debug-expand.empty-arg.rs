use fortuples::fortuples;

struct Container<T>(T);

fortuples! {
    #[tuples::debug_expand()]
    impl Container<#Tuple> {}
}

fn main() {}
