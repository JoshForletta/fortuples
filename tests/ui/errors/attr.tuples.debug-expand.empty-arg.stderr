error: unexpected end of input, expected path
 --> tests/ui/errors/attr.tuples.debug-expand.empty-arg.rs:6:28
  |
6 |     #[tuples::debug_expand()]
  |                            ^
